inlang/source-code/website/src/pages/install/InstallationProvider.tsx

```diff
@@ -140,12 +140,23 @@ async function initializeRepo(
 	/* Opens the repository with lix */
 	const repo = await openRepository(repoURL, {
 		nodeishFs: createNodeishMemoryFs(),
-		corsProxy: publicEnv.PUBLIC_GIT_PROXY_PATH,
+		corsProxy: publicEnv.PUBLIC_GIT_PROXY_BASE_URL + publicEnv.PUBLIC_GIT_PROXY_PATH,
 	})
 
-	const isCollaborator = await repo.isCollaborator({
-		username: user.username,
-	})
+	const isCollaborator = await repo
```

Comment:
```suggestion
	const canCommitToRepository = await repo
```

Reply:
We should probably rename the API in the lix client too. It is not obvious what the scope of `isCollaborator` entails. `canCommit` or `hasCommitRights` is more explicit.

cc @jannesblobel - we discussed this too as part of the telemtry

Reply:
@samuelstroschein  everyone can commit who can read a repo, a collaborator is allowed to push. i think we should change this when naming is clearer and not part of this pr...